content/en/docs/contributing/style-guide.md

```diff
@@ -37,3 +37,64 @@ file.
 
 See also the [Glossary](/docs/concepts/glossary/) for a list of OpenTelemetry
 terms and their definition.
+
+Run `npm run check:text` to verify that all terms and words are written
+properly.
+
+## Markdown standards
+
+To enforce standards and consistency for Markdown files, all files should follow
+certain rules, enforced by
+[markdownlint](https://github.com/DavidAnson/markdownlint). For a full list,
+check the
+[`.markdownlint.json`](https://github.com/open-telemetry/opentelemetry.io/blob/main/.markdownlint.json)
+file.
+
+Run `npm run check:markdown` to verify that all files follow the standard.
+
+Run `npm run fix:format` to fix Markdown related formatting issues.
+
+## Spell checking
+
+Use [CSpell](https://github.com/streetsidesoftware/cspell) to make sure that all
+your text is spelled correctly. For a list of words that are specific to the
+OpenTelemetry website, check the
+[`.cspell.yml`](https://github.com/open-telemetry/opentelemetry.io/blob/main/.cspell.yml)
+file.
+
+Run `npm run check:spelling` to verify that all your words are spelled
+correctly. If `cspell` indicates an `Unknown word` error, verify if you wrote
+that word correctly. If so, add this word to the `cSpell:ignore` section at the
+top of your file. If no such section exists, you can add it to the front matter
+of a Markdown file or as a comment at the top of any other file:
+
+```markdown
+---
+title: PageTitle
+cSpell:ignore: <word>
+---
+```
+
```

Comment:
```suggestion
For any other file, add `cSpell:ignore <word>` in a comment line appropriate for the file's context. For a [registry](/ecosystem/registry/) entry YAML file file, it might look like this:

```